pkgs/jupiter-hw-support/default.nix

```diff
@@ -1,29 +1,81 @@
 { lib
 , stdenv
 , callPackage
-, python3
+, resholve
+, bash
+, coreutils
+, e2fsprogs
+, exfatprogs
+, f3
+, findutils
+, gawk
+, gnused
+, jq
+, parted
+, procps
+, systemd
+, util-linux
 }:
 
 let
   src = callPackage ./src.nix { };
-  pythonEnv = python3.withPackages (py: with py; [
-    evdev
-  ]);
+
+  solution = {
+    scripts = [ "bin/*" "lib/hwsupport/*.sh" ];
+    interpreter = "${bash}/bin/bash";
+    inputs = [
+      coreutils
+      e2fsprogs
+      exfatprogs
+      f3
+      findutils
+      gawk
+      gnused
+      jq
+      parted
+      procps
+      systemd
+      util-linux
+
+      "${placeholder "out"}/lib/hwsupport"
+    ];
+    execer = [
+      "cannot:${e2fsprogs}/bin/mkfs.ext4"
+      "cannot:${procps}/bin/pgrep"
+      "cannot:${systemd}/bin/systemctl"
+      "cannot:${systemd}/bin/udevadm"
+      "cannot:${util-linux}/bin/flock"
+
+      # FIXME: this is a lie. It will run Steam, but it will run it via an executable in ~,
+      # which does not need resholving
```

Comment:
FIXME how?

That sounds more like a `NOTE` than a `FIXME`.

Unless it is possible it causes breakage under some conditions?

Reply:
I don't know, maybe by injecting the steam-but-just-the-protocol-handler instead

Reply:
Added a common protocol handler (and a new FIXME, have fun with that one).